src/App.tsx

```diff
@@ -1,8 +1,33 @@
-import React from 'react';
-
-export const App: React.FC = () => (
-  <div className="App">
-    <p>Nothing was pressed yet</p>
-    <p>The last pressed key is [Enter]</p>
-  </div>
-);
+import { Component } from 'react';
+
+type State = {
+  PressedKey: string,
+};
+
+export class App extends Component<{}, State> {
+  state = {
+    PressedKey: 'Nothing was pressed yet',
```

Comment:
```suggestion
    pressedKey: 'Nothing was pressed yet',
```
Capitalize for components in React